src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -159,6 +159,7 @@
       <mat-card-subtitle class="m-0">{{ selectedVersion?.name }}</mat-card-subtitle>
     </mat-card-header>
     <mat-card-content class="p-3">
+      <div><strong matTooltip="DOI listed in descriptor">DOI</strong>: {{ selectedVersion.doiURL ? selectedVersion.doiURL : 'n/a' }}</div>
```

Comment:
A screenshot in general would be nice. 
This in particular is a bit confusing though, do we ever parse descriptors for DOIs (as opposed to issuing them via zenodo) or is this aspirational?

Reply:
changed matToolTip to "Digital Object Identifier"